eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -59,8 +60,12 @@ function Should-Process-Package($pkgPath, $packageName)
     }
     # Get package info from json file created before updating version to daily dev
     $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
-    Write-Host "SDK Type: $($pkgInfo.SdkType)"
-    return $pkgInfo.IsNewSdk
+    $packagePath = $pkgInfo.DirectoryPath
+    $modifiedFiles = git diff --name-only --relative $TargetBranch HEAD
```

Comment:
Do you care about the deleted one? If you want to filter out the deleted ones. 
Please use `--diff-filter=d` option

Reply:
I need deleted files as well and that's why I didn't include that filter.I need deleted files as well and that's why I didn't include that filter.